book/website/collaboration/hybrid-collab/hybrid-collab-guidelines.md

```diff
@@ -11,60 +11,68 @@
+- Share comprehensive information about the available facilities and support resources and provide guidance on accessing them through facilitation documents.
 - Encourage speakers and attendees who are engaging online to blur their background and to limit background noise as much as possible.
 
 ### Communication Tools for Hybrid Collaboration
 
-- Carefully select tools and technologies that consider inclusivity and the impact on teams.
+- Carefully select tools and technologies considering inclusivity and the impact on teams.
 - Avoid imposing unsuitable platforms on teams and be open to new options and tools.
-- Ensure virtual meetings have the capacity to accommodate more participants.
+- Ensure virtual meetings can accommodate more participants.
 - Follow up with remote attendees after hybrid meetings to address their needs.
-- Set up and use a good quality microphone system for all the in-person groups, so that those joining remotely can clearly hear speakers.
-- Support shared note-taking documents and/ or facilitation guides for both in-person and remote groups to ensure both groups are included.
+- Set up and use a good quality microphone system for all the in-person groups so that those joining remotely can clearly hear speakers.
+- Support shared note-taking documents and/ or facilitation guides for in-person and remote groups to ensure both groups are included.
 - Establish a platform for asynchronous side chat to facilitate communication.
 - Ensure equal availability and access for both in-person and remote attendees in hybrid events.
 
 ### Social Networking during Hybrid Collaboration
 
 - Recognise and address asymmetries in in-person and online settings to establish realistic expectations.
-- Create opportunities for social interactions both online and in person to balance out the challenges.
+- Create opportunities for social interactions both online and in-person to balance out the challenges.
 - Foster a positive atmosphere in person and encourage an informal approach online to facilitate idea exchanges and connections.
 
 ### Personnel and Staffing Planning
 
 Depending on your event or meeting, you may need to consider providing support for session facilitation, notetaking, and technical support. 
-For hybrid events, you will need both in-person personnel **and** online personnel and ensure that are able to recruit support for both the online and in-person parts of your event as it will be incredibly difficult for the same person to act as both an in-person and online facilitator.
+For hybrid events, you will need both in-person personnel **and** online personnel and ensure that can recruit support for both the online and in-person parts of your event as it will be incredibly difficult for the same person to act as both an in-person and online facilitator.
```

Comment:
```suggestion
For hybrid events, you will need both in-person personnel **and** online personnel to support both the online and in-person parts of your event, as it will be incredibly difficult for the same person to act as both an in-person and online facilitator.
```